contracts/claim/src/msg.rs

```diff
@@ -0,0 +1,12 @@
+use schemars::JsonSchema;
+use serde::{Deserialize, Serialize};
+
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema)]
+pub struct InstantiateMsg {}
+
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema)]
+#[serde(rename_all = "snake_case")]
+pub enum ExecuteMsg {
+    #[serde(rename = "claim_mint_nft")]
+    ClaimMintNFT { minter_address: String },
```

Comment:
Without the rename what does serde use?

Reply:
guessing it's claim_mint_n_f_t due to the caps

Reply:
it uses `claim_mint_n_f_t`